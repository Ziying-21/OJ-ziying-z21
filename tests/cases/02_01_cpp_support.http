###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 377
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:56 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:56.870Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 426
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:57 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":125428},{"id":1,"info":"","memory":0,"result":"Accepted","time":58281}],"created_time":"2022-09-09T02:18:56.870Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-09T02:18:57.059Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 406
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:57 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:57.884Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 456
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:58 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":346444},{"id":1,"info":"","memory":0,"result":"Accepted","time":280917}],"created_time":"2022-09-09T02:18:57.884Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C","problem_id":0,"source_code":"#include <stdio.h>\nint main() { printf(\"Hello World!\\n\"); return 0; }","user_id":0},"updated_time":"2022-09-09T02:18:58.517Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"C++","problem_id":0,"source_code":"#include <iostream>\nint main() { std::cout<<\"Hello World!\"<<std::endl; return 0; }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 420
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:58 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:58.897Z","id":2,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"C++","problem_id":0,"source_code":"#include <iostream>\nint main() { std::cout<<\"Hello World!\"<<std::endl; return 0; }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/2 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 419
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:18:59 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Running","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:58.897Z","id":2,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"C++","problem_id":0,"source_code":"#include <iostream>\nint main() { std::cout<<\"Hello World!\"<<std::endl; return 0; }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/2 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 471
# access-control-allow-credentials: true
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:19:00 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":1362071},{"id":1,"info":"","memory":0,"result":"Accepted","time":279244}],"created_time":"2022-09-09T02:18:58.897Z","id":2,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"C++","problem_id":0,"source_code":"#include <iostream>\nint main() { std::cout<<\"Hello World!\"<<std::endl; return 0; }","user_id":0},"updated_time":"2022-09-09T02:19:00.544Z"}
