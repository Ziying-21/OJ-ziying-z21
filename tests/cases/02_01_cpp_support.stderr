[2022-09-09T02:18:55Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:18:55Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:18:56Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000233
[2022-09-09T02:18:56Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 377 "-" "-" 0.000866
[2022-09-09T02:18:57Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 426 "-" "-" 0.000585
[2022-09-09T02:18:57Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 406 "-" "-" 0.001646
[2022-09-09T02:18:58Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 456 "-" "-" 0.000284
[2022-09-09T02:18:58Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 420 "-" "-" 0.001596
[2022-09-09T02:18:59Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/2 HTTP/1.1" 200 419 "-" "-" 0.000221
[2022-09-09T02:19:00Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/2 HTTP/1.1" 200 471 "-" "-" 0.000232
